### ISSUE TITLE
Microsoft Edge (Chromium) - Send_Event process issue

### ISSUE THREAD

**Issue by simonf00** [objective-see/LuLu#153]
I am testing the Microsoft Edge Chromium edition browser. During the install process it runs a process called 'send_event'. It appears that this executable is deleted when the installation process is complete. The problem occurs because Lulu catches it sending something to Microsoft and then requests permission to allow/block. As the update process sometimes runs when I'm not at the machine, the send_event executable has deleted before I get a chance to Allow/Block the communication. This puts the Lulu popup in an infinite loop and cannot be closed. To close it, I have to kill the Lulu Helper process. It also seems to cause chaos for the send_event process itself as it starts using 90% CPU. If I disable Lulu, these issues do not occur.

Any idea how to work around this?

**Comment by simonf00** [comment]
Any info on this... it's a real pain. I have to manually kill the send_event process every time it runs (e.g. every update to the Edge browser). This does not happen if Lulu isn't running.

**Comment by healthyboyv8** [comment]
I have the exact same issue!! it's driving me nuts, I wish there was an automated process that allowed me to catch when send_event is running and kill it.

**Comment by simonf00** [comment]
Has this magically been resolved (either by a change in Edge or macOS)? I still get prompted to Allow/Block, but now I can close the prompt and my CPU isn't getting pinned. Anyone else seeing this?

**Comment by healthyboyv8** [comment]
yeah, I guess you're right. Send_event has not been causing me a problem lately.
